### PR TITLE
Unauthenticated access vulnerability to crAPI

### PR DESCRIPTION
**Pull request by mathew-jose**
## Description
Added unauthenticated access vulnerability to the get orders endpoint by removing JWT token check and the check to whether the user has access to view the order.

### Testing
Tested the endpoint on Postman without JWT token. 

### Documentation
Added the new challenge to the documentation.

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged
- [x] I have documented any changes if required in the [docs](docs).

### CODE REVIEW

**Comment by piyushroshan** [comment]
Please add a test case for the same.

**Comment by mathew-jose** [comment]
Unit test cases are added.

File: services/workshop/crapi/shop/tests.py [hunk 1]
```diff
@@ -163,3 +166,35 @@ def test_sql_injection(self):
...
+        order_details = {
+            "product_id":  str(self.product_id),
+            "quantity": 1
+          
+        }
+        res = self.client.post('/workshop/api/shop/orders', order_details, content_type='application/json', **self.auth_headers)
+        self.order_id = json.loads(res.content)['id']
+        self.assertEqual(res.status_code, 200)
+
+    def Test_unauthenticated_get_order(self):
```
Comment (piyushroshan) [inline services/workshop/crapi/shop/tests.py:184]: Remove Test_ from the non test case. Helper functions should not start with Test_

Comment (mathew-jose) [reply services/workshop/crapi/shop/tests.py]: Noted, I have removed them.

**Review by piyushroshan — APPROVED** [review]

**Pull request merged**